### PR TITLE
feat(alerts): Create GitHub Enterprise alert rule action

### PR DESCRIPTION
**Pull request by schew2381**
Followup to https://github.com/getsentry/sentry/pull/58175

Create an alert rule action to create an Issue on GitHub Enterprise, matching the GitHub alert rule action

### CODE REVIEW

File: src/sentry/integrations/github/actions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from .create_ticket import GitHubCreateTicketAction
+
+__all__ = ("GitHubCreateTicketAction",)
```
Comment (ceorourke) [inline src/sentry/integrations/github/actions/__init__.py:3]: How was this not already here if we have GH issue creation from rules?

File: tests/sentry/integrations/github_enterprise/test_ticket_action.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        return ExternalIssue.objects.get_linked_issues(event, self.integration).values_list(
+            "key", flat=True
+        )[0]
+
+    @responses.activate()
+    def test_ticket_rules(self):
+        title = "sample title"
+        sample_description = "sample bug report"
+        html_url = f"https://github.example.com/api/v3/foo/bar/issues/{self.issue_num}"
+
```
Comment (ceorourke) [inline tests/sentry/integrations/github_enterprise/test_ticket_action.py:80]: I'm surprised we don't need a response like this: https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/github_enterprise/test_issues.py#L117-L121 but I haven't worked w/ GHE much

**Comment by ceorourke** [comment]
Overall it looks good, 2 questions - why were you sitting on this since November and did you get GHE set up locally to test it? If not, it might be safer to put behind a flag.

File: src/sentry/integrations/github/actions/__init__.py (hunk 1 above)
Comment (schew2381) [reply src/sentry/integrations/github/actions/__init__.py:3]: I think I forgot to do this in the original github pr. I believe it's only used for importing

**Comment by schew2381** [comment]
> Overall it looks good, 2 questions - why were you sitting on this since November and did you get GHE set up locally to test it? If not, it might be safer to put behind a flag.

I waited a while b/c we don't have enterprise setup locally, but at this point it seems more practical to release this behind a flag/option.

File: tests/sentry/integrations/github_enterprise/test_ticket_action.py (hunk 2 above)
Comment (schew2381) [reply tests/sentry/integrations/github_enterprise/test_ticket_action.py:80]: 🤞

Comment (schew2381) [reply tests/sentry/integrations/github_enterprise/test_ticket_action.py:80]: Wait turns out I am doing it lol: https://github.com/getsentry/sentry/pull/59315/files#diff-48d5259d445171b0a9fc4a47007fb56b42dcee4decf40b649f3b2b97f6ba2bddR56-R59

**Review by ceorourke — APPROVED** [review]
Looks good, I think you just need to add the flag to the failing tests

**Pull request merged**